framework/data/Sort.php

```diff
@@ -301,9 +308,7 @@ public function setAttributeOrders($attributeOrders, $validate = true)
      */
     public function getAttributeOrder($attribute)
     {
-        $orders = $this->getAttributeOrders();
-
-        return isset($orders[$attribute]) ? $orders[$attribute] : null;
+        return ArrayHelper::getValue($this->getAttributeOrders(), $attribute);
```

Comment:
It also can break code that uses related attributes, `author.name` will evaluate differently in new and old code.